client/state/ui/editor/post/actions.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * External dependencies
+ */
+import debugFactory from 'debug';
+
+const debug = debugFactory( 'calypso:ui:editor:post:actions' );
+
+export function setEditingMode( mode, modeTitle, site ) {
+	debug( 'setEditingMode', mode, modeTitle, site );
+	return { type: 'TODO' };
+}
+
+export function startEditingNew( site, postOptions ) {
+	debug( 'startEditingNew', site, postOptions );
+	return { type: 'TODO' };
+}
+
+export function startEditingExisting( site, postID ) {
+	debug( 'startEditingExisting', site, postID );
+	return { type: 'TODO' };
+}
+
+export function toggleStickyStatus( currentStatus ) {
```

Comment:
Is there some discussion to why these actions have been split out to individual properties, than our [existing approach](https://github.com/Automattic/wp-calypso/blob/3c8fb4f98ecc713370ebbb9feaa5883321f60b33/client/lib/posts/actions.js#L237-L247) where we accept an object of updated attributes?